war/src/main/js/widgets/config/tabbar.less

```diff
@@ -92,9 +83,226 @@
+          padding-right:25px;
+        }
+        
+        select:not([multiple]){
+          height:@input-line-height;
+          line-height:@input-line-height;
+        }
+        select{
+          border-radius:0;
+          background:@brightest;
+        }
+        .help{
+          background:@medium-translucent;
+        }
+        p,
+        .help,
+        .setting-description {
+          font-size:1em;
+          line-height:1.4;
+        } 
+      }
+      td[colspan="4"]:empty{
+        display:none
+      }
+      td.setting-help{
+        vertical-align:top;
+        padding:5px 10px 5px 5px
+      }
+      td.setting-leftspace:after {
+        content:' '; 
+        display:block; 
+        width:13px; 
+        height:1px;
+      }
+      .repeated-chunk,
+      tr[nameref^="rowSetStart"],
+      tr[nameref^="cb"],
+      tr[nameref^="radio-block"] {
+        background:@shade-hint; 
+        border-left:5px solid @shade
+      }
+      .hetero-list-container,
+      .repeated-container,
+      tr.optional-block-start,
+      tr.radio-block-start {
+        border-left:5px solid @shade; 
+        background:none;
+      }
+      tr.help-area {
+        background:none !important;
+        
+        td {
+          padding:0 5px
+        }
+      }
+      .CodeMirror {
+        background:@brightest;
+  
+      }
+      .CodeMirror-scroll {
+          border:1px solid @solid-border !important;
+      }
+      .repeated-container > .repeated-chunk {padding-top:15px;}
+      .repeated-chunk.hover{ 
+        border-color:@line-blue; 
+        box-shadow:0 2px 10px @shade, inset 0 200px 200px -200px @brightest; 
+        position:relative; 
+        z-index:2
+      }
+      .repeated-chunk.dragging{
+        position:static;
+      }
+      .repeated-chunk{
+        border:1px solid @shade;
+        margin:2px;
+        position:relative;
+      }  
+      .repeatable-delete {
+        position:absolute;
+        top:-2px;
+        right:35px;
+        height:18px;
+        width:32px;
+        border-bottom-left-radius:@radius;
+        border-bottom-right-radius:@radius;
+        overflow:hidden;
+        
+        button{
+          padding:0;
+          margin:0;
+          text-indent:999em;
+          background-color: @danger;
+          color: @light-backgrond;
+          border: 1px solid @danger-line;
+        }
+        button:hover{
+          background-color: @danger-dark;
+          border-color:@danger-dark-line;
+        }
+        button:before{
+          content: 'X';
+          font-weight: bold;
+          text-indent: 0;
+          display: block;
+          position: absolute;
+          left: 0;
+          right: 0;
+          top: 0;
+          bottom: 0;
+          text-align: center;
+          font-size: 12px;
+          line-height: 18px;
+        }  
+  
+      }
+    }
+    span.highlight {
+      background-color: #ffff00;
+    }
   }
-}
\ No newline at end of file
+}
+#ygddfdiv {
```

Comment:
Gus, Is there really something with this ID ?  Might be worth adding a comment about it because it looks like was a mistake.